ca_description/meshes/LICENSE_create_1.txt

```diff
@@ -0,0 +1,31 @@
+Software License Agreement (BSD License)
```

Comment:
I didn't see this particular 3-clause license in the *create_description* package of the [turtlebot_create](https://github.com/turtlebot/turtlebot_create) repo. I think it will suffice to add a link in the README (as I have for the existing URDF and mesh files) instead of including this file. I will probably make the README a little more verbose.